src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -891,69 +858,63 @@ class BasicLayout extends React.PureComponent{
 
                 </div>
                 <div className="modal-backdrop fade show"> </div>
-            </React.Fragment>
-        }
-        else{
-            return
+            </React.Fragment>;
         }
     }
     getFormButtons(){
         //need to check if form is ready, if not just add the cancel button, or disable the create?
         return <React.Fragment>
             <button type="button" className="btn btn-secondary" data-dismiss="modal" onClick={this.onCancelNewWidget}>Cancel</button>
-            <button type="button" className="btn btn-primary" onClick={this.onCreateNewWidget} disabled={!this.state.FormReady}>Create</button>
-        </React.Fragment>
+            <button type="button" className="btn btn-primary" onClick={this.onCreateNewWidget} disabled={!this.state.formReady}>Create</button>
+        </React.Fragment>;
 
     }
     getBaseMapSelector(){
-        if(this.state.selectedWidgetType == "ImageCollection" || this.state.selectedWidgetType == "DualImageCollection" || this.state.selectedWidgetType == "imageAsset" || this.state.selectedWidgetType == "imageCollectionAsset") {
+        if(this.state.selectedWidgetType === "ImageCollection" || this.state.selectedWidgetType === "DualImageCollection" || this.state.selectedWidgetType === "imageAsset" || this.state.selectedWidgetType === "imageCollectionAsset" || this.state.selectedWidgetType === "ImageElevation") {
```

Comment:
["ImageCollection", "DualImageCollection", "imageAsset", "ImageElevation"].includes(this.state.selectedWidgetType)